apps/general/serializers.py

```diff
@@ -1 +1,41 @@
-#Create Serializers
\ No newline at end of file
+from rest_framework import serializers
+from .models import *
+from apps.master.serializers import *
+
+#Create Serializers
+        
+class LedgerAccountsSerializers(serializers.ModelSerializer):
+    ledger_group = Mod_LedgerGroupsSerializer(source='ledger_group_id', read_only=True)
+    class Meta:
+        model = LedgerAccounts
+        fields = '__all__'
+
+class Mod_LedgerAccountsSerializers(serializers.ModelSerializer):
```

Comment:
This class name should be in camelCase. PLease change

Reply:
Updated